### DISCUSSION TITLE
M2A collection configuration through regex

### DISCUSSION THREAD

**Discussion by scolastico** [directus/directus#12517] [Feature Requests]
I think it could be very useful if it were possible in an M2A collection to allow any collection that matches a regex pattern. For example, if I have a page with an M2A relationship to multiple databases of items, I could define that the M2A allows any database ending in, say for example, "_item".

**Comment by rijkvanzanten** [comment]
Love it!

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
